assets/messages.yml

```diff
@@ -1,3 +1,14 @@
+categories:
+  - category: duplicate
+    value: Duplicate
```

Comment:
Not so important, but maybe `name` or similar would be more expressive than just `value` (I assume you chose `value` because this was needed when those were still variables).

Reply:
Ah yeah, I chose it for consistency. I'll change that.